### PR TITLE
Add Damage Command

### PR DESCRIPTION
**Pull request by Nitrogen-Dioxide**
And add isNumeric() mehod to know if the string is numeric.

Well I find a method from am0d hacked client to damages you in **multiplayer only**.
The amount of damage goes trough 0 and 40.

### CODE REVIEW

File: Wurst Client/src/tk/wurst_client/commands/DamageCmd.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@Cmd.Info(help = "Damages you with given ammount.", name = "damage", syntax = {"[ammount]"})
+public class DamageCmd extends Cmd
+{
+	@Override
+	public void execute(String[] args) throws Error
+	{
+        if (args.length == 0)
+        	syntaxError();	
+        if (!isNumeric((args[0])))
+        	syntaxError("Ammount must be a number.");
```
Comment (pokeball99) [inline Wurst Client/src/tk/wurst_client/commands/DamageCmd.java]: amount is spelled wrong

File: Wurst Client/src/tk/wurst_client/commands/DamageCmd.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        	error("Cannot damage when in singleplayer.");
+        else if (dmg >= 0 && dmg < 40)
+        
+        {	if (!mc.thePlayer.isOnLadder() && mc.thePlayer.onGround && !mc.thePlayer.capabilities.isCreativeMode)
+        mc.getNetHandler().addToSendQueue(GroundFalse);
+        mc.getNetHandler().addToSendQueue(setDamage);
+        mc.getNetHandler().addToSendQueue(GroundTrue);
+        }
+		else
+			syntaxError("Ammount is too low or too high.");
```
Comment (pokeball99) [inline Wurst Client/src/tk/wurst_client/commands/DamageCmd.java]: here too

**Comment by Nitrogen-Dioxide** [comment]
@pokeball99 xD I'm feeling stupid.

**Comment by pokeball99** [comment]
:D

**Comment by Alexander01998** [comment]
Merged into `PR#540-damage-cmd` for editing & testing.

**Comment by Alexander01998** [comment]
Didn't do anything. Tested with & without NoCheat+, with & without NoFall.

**Comment by Nitrogen-Dioxide** [comment]
@Alexander01998 
The place that you used damage, have fall damage enabled?

**Comment by Nitrogen-Dioxide** [comment]
It is working fine to me. *On places that you can fall damage*.

untitled_000075
untitled_000121
untitled_000395

And how did you make that sweet gif. And why the letters in the chat are colorful xD

**Comment by Alexander01998** [comment]
After doing some more tests, it seems to work on servers with NoCheat+ and fall damage. It actually took me a while to find a server that had both.

> And how did you make that sweet gif.

I used [ScreenToGif](https://screentogif.codeplex.com/).

> And why the letters in the chat are colorful xD

A GIF image can only have 256 different colors. The water, sky, stone, trees, houses, etc. took them all away before the chat could use them.

**Comment by Nitrogen-Dioxide** [comment]
@Alexander01998 You can test on mineplex too. There are a lot of mini-games that have fall damage.

**Pull request merged**